noetic/distribution.yaml

```diff
@@ -75,6 +75,12 @@ repositories:
       url: https://github.com/ros/actionlib.git
       version: noetic-devel
     status: maintained
+  adi_3dtof_image_stitching:
+    source:
+      type: git
+      url: https://github.com/analogdevicesinc/adi_3dtof_image_stitching.git
```

Comment:
There are binary libraries checked into this source repository.  There are two general problems with this:

1. Our usual practice isn't to have binaries checked into the repository, but rather to have the source there and to build it on the buildfarm.
2. There is no license listed listed for those binary files, so we don't know if we are allowed to redistribute them.

Can you comment on these binary libraries?

Reply:
The library file corresponds to some of our codes which are undergoing internal review. We will replace the library with the source files after the review is complete. 
The license for this library is the same as the License for the main repo which is in the root directory(BSD-3).

Reply:
OK, thanks for the response.

In that case I think we should hold off on this PR until those binaries are replaced with the sources.